CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 `Unreleased`_
 =============
 
+Fixed
+-----
+* Made BlueZ callback logging lazy to improve performance.
```

Comment:
It would be more accurate to say `BlueZ D-Bus signal callback` since we haven't done this in general for all callbacks.

Reply:
Squashed and repushed